utils/installer/install.ps1

```diff
@@ -0,0 +1,155 @@
+#Set Variable to master is not set differently
+if ($null -eq $LVBRANCH) {
+    $LVBRANCH = "master"
+}
+
+#set -o nounset # error when referencing undefined variable
+$ErrorActionPreference = "Stop" # exit when command fails
+
+function mkdirondemand($path) {
+	if ((Test-Path $path) -eq $false) {
+		mkdir $path
+	}
+}
+
+function commandinstalled($cmd){
+	try { Get-Command $cmd > $null }
```

Comment:
I would use `-ErrorAction` instead, so maybe it would look like this:

```pwsh
function verifyInstalled($cmd) {
    try {
        Get-Command -Name $cmd -ErrorAction Stop | Out-Null
    }
    catch { "Error! command [$cmd] is not found, please install and try again." }
}
```